lib/main_common.pm

```diff
@@ -488,7 +488,7 @@ sub snapper_is_applicable {
 }
 
 sub chromestep_is_applicable {
-    return check_var('ARCH', 'i586') || check_var('ARCH', 'x86_64');
+    return is_opensuse && (check_var('ARCH', 'i586') || check_var('ARCH', 'x86_64'));
```

Comment:
we only test on x86 on o3 so why not just `return is_opensuse`?

Reply:
@nicksinger Simply because you are wrong. Open a browser and go to openqa.opensuse.org and you should see !x86_64 as well.

Reply:
sry, brain lag. You're right :+1: